internal/routes/server.go

```diff
@@ -393,6 +400,16 @@ func realmkeysRoutes(r *mux.Router, c *realmkeys.Controller) {
 	r.Handle("/keys/activate", c.HandleActivate()).Methods("POST")
 }
 
+// realmSMSkeysRoutes are the realm key routes.
+func realmSMSkeysRoutes(r *mux.Router, c *smskeys.Controller) {
+	r.Handle("/smskeys", c.HandleIndex()).Methods("GET")
```

Comment:
Can we use `/sms-keys` instead please? It's a friendlier route. Will have to update any href links.